content/api/commands/origin.md

```diff
@@ -473,49 +473,114 @@ of
 [restrictions on the data which may be passed](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm#things_that_dont_work_with_structured_clone)
 into the callback.
 
-### Callback restrictions
+### Dependencies / Sharing Code
 
-Because of the way in which the callback is transmitted and executed, there are
-certain limitations on what code may be run inside it. In particular, the
-following Cypress commands will throw errors if used in the callback:
+It is not possible to use
+[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+or
+[dynamic ES module `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
+within the callback. However, [`Cypress.require()`](/api/cypress-api/require)
+can be utilized to include [npm](https://www.npmjs.com/) packages and other
+files. It works exactly the same as
```

Comment:
At least as far as a `require` used in browser-targeted code, there is no difference, because it just gets replaced with `require` anyways. There _could_ technically be differences between the `require` that webpack implements and CommonJS `require`, so maybe we should hedge a little bit on the language to give us some wiggle room there. Mainly what I'm trying to communicate is that users can think of it as if they're using `require` anywhere else in their spec code. I'll change up the wording to communicate that better.

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4681/commits/b6685be47aad42384813fb3f48e3e39828a519a6